main/src/test/scala/ParseKey.scala

```diff
@@ -7,26 +7,47 @@
 
 package sbt
 
-import java.net.URI
-
-import org.scalacheck.Arbitrary.{ arbBool, arbitrary }
-import org.scalacheck.Gen._
-import org.scalacheck.Prop._
-import org.scalacheck._
 import sbt.Def.{ ScopedKey, displayFull, displayMasked }
-import sbt.internal.TestBuild._
-import sbt.internal.util.AttributeKey
-import sbt.internal.util.complete.{ DefaultParsers, Parser }
-import sbt.internal.{ Resolve, TestBuild }
-import sbt.librarymanagement.Configuration
+import sbt.internal.TestBuild2._
+import sbt.internal.util.complete.Parser
+import sbt.internal.{ Resolve, TestBuild2 }
+import hedgehog.{ Result => Assert, _ }
```

Comment:
@eed3si9n The only reason for the rename is that `sbt` exposes a `Result` object as well. I'm not sure if there's a better/different way to exclude that from `sbt` instead of this rename?

Reply:
Generally speaking I think putting test cases in `sbt` might not be a good practice since it gives access to the internals. Sometimes necessary, but I don't know if that should be the default.

Reply:
Changing to another package didn't really work because we are using stuff from package private..

```
[error] /Users/eed3si9n/work/sbt-modules/sbt/main/src/test/scala/ParseKey.scala:150:37: method fromConfigIdent in trait KeyIndex cannot be accessed in sbt.internal.KeyIndex
[error]       val name = structure.keyIndex.fromConfigIdent(maybeResolvedProj)(configID)
[error]                                     ^
[error] /Users/eed3si9n/work/sbt-modules/sbt/main/src/test/scala/ParseKey.scala:174:29: method displayPedantic in object Scope cannot be accessed in object sbt.Scope
[error]         .log(s"Key: ${Scope.displayPedantic(key.scope, key.key.label)}")
[error]                             ^
```

Reply:
Fully qualified `hedgehog.Result` I think is better than renaming this to `Assert`.

Reply:
> Fully qualified `hedgehog.Result` I think is better than renaming this to `Assert`

👍 Sounds good. Agreed.